helios-docs/docs/QuickStart.md

```diff
@@ -0,0 +1,64 @@
+# QuickStart
+
+Once added helios to you project, let's start to add the needed imports:
+
+```kotlin:ank:silent
+import arrow.core.*
+import helios.core.*
+import helios.meta.*
+import helios.optics.*
+import helios.typeclasses.*
+```
+
+Now, we can start to create our DSL.
+
+## DSL
+
+```kotlin
+@json
+data class Person(val name: String, val age: Int) {
+  companion object
+}
+```
+
+The `@json` annotation will provide the decoder and encoder for that data class, so we are able to
```

Comment:
This sentence seems to be missing the end. Perhaps ", so we are able to read from and write to Json." ?